int8 finger
bool high_friction
---
bool success
